050-Manage/060-migrations-multischema.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Multi Schema Migrations
+navTitle: Multi Schema Migrations
```

Comment:
```suggestion
navTitle: Multi schema migrations
```

Reply:
resolving for now because of the new name (multi-version)